dockerfiles/base/tests/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM docker:1.12.6-dind
+
+RUN apk add --no-cache bash
+
+RUN curl -o /tmp/v0.4.0.tar.gz -L https://github.com/sstephenson/bats/archive/v0.4.0.tar.gz \
+	&& tar -x -z -f /tmp/v0.4.0.tar.gz -C /tmp \
+	&& bash /tmp/bats-0.4.0/install.sh /usr/local \
+	&& rm -rf /tmp/*
+
+WORKDIR /home/testing
```

Comment:
there is a missing Header in the file
Also we should include a build.sh script to build it